feat: openapi typescript client (#2885)
Redocly/redocly-cli: docs/@v2/configuration/reference/apis.md

- Output file path
- When running `bundle` without specifying an API, the bundled API description is saved to this location.

---

- client
- [Client object](./client.md)

> - Additional TypeScript client generation configuration for this API, used by the experimental `generate-client` command.

Reviewer: ```suggestion
- TypeScript client generation configuration override for this API, used by the `generate-client` command.
```

See this comment: https://github.com/Redocly/redocly-cli/pull/2885/changes#r3666819715.

---

> - clientOutput

Reviewer 2: should the output be inside of client?

Author: not really, we wanted to follow existing approach of config, discusssed with @tatomyr

- Output file path

> - When running `generate-client` without specifying an API, the generated TypeScript client is saved to this location.

Reviewer: ```suggestion
- The generated client is saved to this location unless `--output` overrides it.
```
